docs/components/examples/Asset.tsx

```diff
@@ -55,7 +55,7 @@ export const Asset = () => {
     isDragReject,
   } = useDropzone({
     accept: {
-      'video/*': ['*.mp4'],
+      'video/*': ['.mp4'],
```

Comment:
Could you also make this change here as well?

https://github.com/livepeer/livepeer.js/blob/2f1b286f65a2eda0b0ff0c2142ec8744d23a4dab/docs/pages/examples/react/view-asset.en-US.mdx

Thanks!

Reply:
@0xcadams, see below 👇